db/schema.rb

```diff
@@ -23,18 +23,60 @@
     t.datetime "image_updated_at"
   end
 
+  create_table "admin_admission_eligibilities", force: :cascade do |t|
+    t.string   "name"
+    t.text     "description"
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
+  end
+
   create_table "admin_departments", force: :cascade do |t|
     t.string   "name"
     t.string   "short_form"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
 
+  create_table "admin_notice_boards", force: :cascade do |t|
+    t.string   "name"
+    t.string   "notice_type"
+    t.integer  "semester_id"
+    t.integer  "department_id"
+    t.datetime "created_at",    null: false
+    t.datetime "updated_at",    null: false
+    t.index ["department_id"], name: "index_admin_notice_boards_on_department_id"
+    t.index ["semester_id"], name: "index_admin_notice_boards_on_semester_id"
+  end
+
+  create_table "admin_notice_forms", force: :cascade do |t|
+    t.string   "description"
+    t.string   "background_color"
+    t.integer  "notice_board_id"
+    t.datetime "created_at",       null: false
+    t.datetime "updated_at",       null: false
+    t.index ["notice_board_id"], name: "index_admin_notice_forms_on_notice_board_id"
+  end
+
   create_table "admin_semesters", force: :cascade do |t|
     t.string   "name"
     t.string   "short_form"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
 
+  create_table "admin_tuition_fees", force: :cascade do |t|
```

Comment:
Why it is render ? Is this story told you to make `admin_tuition_fees` table ? Please remove this un necessary table